Sources/Networking/Operations/Handling/SubscriberAttributesMarshaller.swift

```diff
@@ -1,27 +0,0 @@
-//
-//  Copyright RevenueCat Inc. All Rights Reserved.
-//
-//  Licensed under the MIT License (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//      https://opensource.org/licenses/MIT
-//
-//  SubscriberAttributesMarshaller.swift
-//
-//  Created by Joshua Liebowitz on 11/18/21.
-
-import Foundation
-
-enum SubscriberAttributesMarshaller {
-
-    // fixme: make `SubscriberAttributeDict` `Encodable` instead
```

Comment:
This was the goal of this PR, but as I explained in the description it didn't make sense. At least the existing implementation is cleaner now.